wandb/sdk/wandb_run.py

```diff
@@ -949,7 +953,13 @@ def log_code(
         Returns:
             An `Artifact` object if code was logged
         """
-        name = name or "{}-{}".format("source", self._run_id)
+        name = name
```

Comment:
why?

Reply:
Great question, if every job artifact is based on a run id, than no runs will share a single job artifact, making jobs produce from artifacts redundant and less valuable.

Reply:
i mean you already have name variable why assign it to another variable called name? :)

Reply:
lmao oops